### ISSUE TITLE
[fr] suggest "t'es" (colloquial) > "tu es"

### ISSUE THREAD

**Issue by jaumeortola** [languagetool-org/languagetool#3502]
Different rules are needed for:
* je ne t'comprends > je ne te comprends
* t'comprends, t'es > tu comprends, tu es...

**Comment by udomai** [comment]
I agree, the case _je (ne) t'comprends pas_ si a pretty unequivocal context and can get a rule for itself.

"t'comprends", or, with -er verbs, "t'critique", can be a misspelling of:

- tu comprends/critiques
- te comprends/critique/critiques
- te comprend/critique/critiquent

That rule is a doozy! The context will be quite hard to sort out since people often misspell homophonous verb forms. When in doubt, better suggest all the possibilities. But, of course, we must differentiate the rule as much as can be done.

**Comment by jaumeortola** [comment]
Also to do:
* T'inquiètes pas (colloquial) > Ne t'inquiètes pas.
